File: CS/MultiSelectColumnCustomization/CustomGridControl/GridInfoRegistratorOverride.cs
using DevExpress.XtraGrid;
using DevExpress.XtraGrid.Registrator;
using DevExpress.XtraGrid.Views.Base;

namespace MultiSelectColumnCustomization {
	public class GridInfoRegistratorOverride : GridInfoRegistrator {
		public override BaseView CreateView(GridControl grid) {
			return new GridViewOverride(grid as GridControl);
		}

		public override string ViewName {
			get {
				return "GridViewOverride";
			}
		}
	}
}

File: VB/MultiSelectColumnCustomization/CustomGridControl/GridInfoRegistratorOverride.vb
Imports DevExpress.XtraGrid
Imports DevExpress.XtraGrid.Registrator
Imports DevExpress.XtraGrid.Views.Base

Namespace MultiSelectColumnCustomization

    Public Class GridInfoRegistratorOverride
        Inherits GridInfoRegistrator

        Public Overrides Function CreateView(ByVal grid As GridControl) As BaseView
            Return New GridViewOverride(TryCast(grid, GridControl))
        End Function

        Public Overrides ReadOnly Property ViewName As String
            Get
                Return "GridViewOverride"
            End Get
        End Property
    End Class
End Namespace
